src/generator.cpp

```diff
@@ -553,8 +553,8 @@ struct Generator::GeneratorImpl
 
     Generator::ModelType mModelType = Generator::ModelType::UNKNOWN;
 
-    std::list<GeneratorInternalVariablePtr> mInternalVariables;
-    std::list<GeneratorEquationPtr> mEquations;
+    std::list<GeneratorInternalVariablePtr> mInternalVariables = {};
+    std::list<GeneratorEquationPtr> mEquations = {};
```

Comment:
This shouldn't be needed...?

Reply:
No it's not I thought clang tidy might remind me to remove it.